app/static/js/userProfile.js

```diff
@@ -216,8 +216,13 @@ $(document).ready(function(){
              "noteTextbox": $("#addNoteTextArea").val(),
              "bonner": isBonner ? "yes" : "no"},
       success: function(response) {
+        msgToast("Successfully added a note", "", 2000)
         target = isBonner ? "bonner" : "notes"
-        reloadWithAccordion(target)
+        // Delay reload to let user see the message
+            setTimeout(function() {
+                reloadWithAccordion(target)
+                location.reload()
+            }, 2500) 
```

Comment:
Shouldn't this be using the new flash that shows up on the next page reload instead of a toast?